File: subprojects/client/src/main/groovy/com/canoo/dolphin/core/client/comm/OnFinishedHandlerAdapter.java
package com.canoo.dolphin.core.client.comm;

import com.canoo.dolphin.core.client.ClientPresentationModel;

import java.util.List;
import java.util.Map;

public class OnFinishedHandlerAdapter implements OnFinishedHandler {
    @Override
    public void onFinished(List<ClientPresentationModel> presentationModels) {
        // do nothing
    }

    @Override
    public void onFinishedData(List<Map> data) {
        // do nothing
    }
}

File: subprojects/client-javascript/js/dolphin/OnFinishedHandlerAdapter.ts
import ofn = require("../../js/dolphin/OnFinishedHandler");
import pm  = require("../../js/dolphin/ClientPresentationModel");
export module dolphin {

    export class OnFinishedHandlerAdapter implements ofn.dolphin.OnFinishedHandler {
        onFinished(presentationModels:pm.dolphin.ClientPresentationModel[]):void {
        }

        onFinishedData(data:any[]) {

        }

    }
}